generators/server/templates/app/src/main/resources/application.properties

```diff
@@ -5,6 +5,9 @@ spring.main.allow-bean-definition-overriding=true
 spring.jmx.enabled=false
 spring.mvc.problemdetails.enabled=true
 
+################ Logging #####################
+logging.file.name=logs/${spring.application.name}.log
```

Comment:
As we are configuring everything related to logging in logback-spring.xml we can configure log filename also there as follows:

```
<springProfile name="!default">
        <property name="LOG_FILE" value="app.log"/>
        <include resource="org/springframework/boot/logging/logback/file-appender.xml" />
        <root level="INFO">
            <appender-ref ref="FILE" />
            <appender-ref ref="CONSOLE" />
        </root>
</springProfile>
```

Reply:
Addressed.